lib/aamva/request/security_token_request.rb

```diff
@@ -121,7 +122,7 @@ def uuid
       end
 
       def timeout
-        ENV.fetch('AAMVA_AUTH_REQUEST_TIMEOUT', 5).to_i
+        (config.auth_request_timeout || 5).to_i
```

Comment:
same here, I think we ideally set up the config as much as possible at the beginning?